README.md

```diff
@@ -14,10 +18,41 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+##### The requirements for running this application are Node.js 4.4.4 and npm 3.8.9.
+You should use Node Version Manager (nvm) to manage the versions of node.js on your local machine.
+To install please visit: https://github.com/creationix/nvm
+_If you are a on a mac and use [homebrew](http://brew.sh/) you can install nvm by typing: brew update && brew install nvm_
```

Comment:
Typo: extra `a` after `are`, should read: `If you are on a mac`. Also should be a comma after the homebrew link: `If you are on a mac and use homebrew,`.